docs/Dockerfile

```diff
@@ -6,31 +6,17 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 # General requirements not in the base image
 RUN apt-get -qqy update \
-  && apt-get -qqy install \
-    ca-certificates \
-    curl \
-    git \
-    npm \
-    python3 \
-    python3-distutils \
-    python3-pip \
-    python3.9 \
-    python3-distutils \
-    python3.9-venv \
-    software-properties-common \
-    tzdata \
-    sudo \
-    unzip \
-  # Set Python 3.9 as the default
-  && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1 \
-  && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 2;
-
-# Make sure we're using the latest pip
-RUN pip install --upgrade pip \
-  && pip install virtualenv
+  && apt-get -qqy install git npm python3.9-venv
 
 WORKDIR /app/
 
-COPY ./package.json ./
+COPY package.json requirements.txt .
+
 RUN npm install .
 ENV PATH=/app/node_modules/.bin:$PATH
+
+# Inspired by https://pythonspeed.com/articles/activate-virtualenv-dockerfile/
```

Comment:
Could we just write a comment directly explaining what's going on rather than pointing at a URL?

I was going to suggest we add a `--no-venv` option to `wpt` because containing everything in a Dockerfile and then adding a second level of containment with a virtualenv just seems somewhat silly. The problem is that although that works well enough to start generating the docs, we call `create_complete_parser` in `wpt.py` when generating the command line docs. That tries to install all the dependencies for all commands, and the setup there assumes that we have a virtualenv rather than just a path to a pip binary. We could refactor that of course, but it might be more effort than it's worth.

Reply:
I also thought it was silly to put a virtualenv inside of a docker image, but then it should ensure that we don't depend on system Python libraries that happen to be in the image, which is good, even if one could "just" fix that. And yes, I've wished for a `--no-venv` argument, but couldn't be bothered :)